queenbee/repository/package.py

```diff
@@ -274,12 +274,12 @@ def from_package(cls, package_path: str):
 
     def fetch_package(self, source_url: str = None, verify_digest: bool = True, auth_header: str = '') -> 'PackageVersion':
         if source_url.startswith('file:'):
-            source_path = source_url.split('file:')[1]
-
+            source_path = source_url.split('file:///')[1]
+            subfolder = f'{self.type}s'
             if os.path.isabs(source_path):
-                package_path = os.path.join(source_path, self.url)
+                package_path = os.path.join(source_path, subfolder, self.url)
```

Comment:
We don't need to add the `subfolder` here. The `self.url` is be a valid relative path from the `source_path`.

![image](https://user-images.githubusercontent.com/20436635/94407844-e92b7580-016b-11eb-9c83-5983c7bdfd45.png)

Reply:
This is very different from what I get for a local repository:

![image](https://user-images.githubusercontent.com/2915573/94448689-2f3b0600-0179-11eb-883e-31cb6178a5fc.png)

We need the subfolder to get the local ones to work.

Reply:
This commit should fix the bug for adding subfolder name to the url.

https://github.com/ladybug-tools/queenbee/pull/148/commits/e4928355a29aafa2832f8b0b6b961dac0d52e083